docs/site/repository/gitdev/index.md

```diff
@@ -0,0 +1,567 @@
+release but not the current release.
+
+You may need to add "DISABLE_JIRA_ISSUE_MATCH=true" and/or
+"ALLOW_MANY_COMMITS=true" to the PR description to silence errors coming from
+the Unicode bot.
+
+You should use a MERGE COMMIT to merge from maint to main, NOT REBASE MERGE as
+is normally recommended. You will need to go into the admin panel on GitHub,
+enable merge commits, perform your merge commit, and then disable merge commits
+again from the admin panel. When making your merge commit, remember to use the
+correct commit message syntax: prefix the merge commit message with ICU-#####,
+the new ticket number you created above.
+
+**The Hard Way (Merge Conflicts):** At the end of the day, the goal is that main
+should share the maint branch's history. This is done using merge commits. What
+follows is an example of how to create merge commits that retain full branch
+history.
+
+Create a new branch based on the tag you want to merge:
+
+```
+git fetch upstream
+git checkout main
+git checkout -b 64-merge-branch  # use any name you like
+```
+
+*If you already have this branch from a previous release tag*, you could either
+use a new branch, or merge the latest main into your branch:
+
+```
+git checkout 64-merge-branch
+# DANGER: Please make sure your workspace is clean before proceeding!
+# If it's not, you might sneak in unreviewed changes.
+git merge --no-commit main
+git commit -am "ICU-##### Merge tag 'main' into 64-merge-branch"
+```
+
+Now, merge in maint:
+
+```
+# DANGER: Please make sure your workspace is clean before proceeding!
+# If it's not, you might sneak in unreviewed changes.
+git merge --no-commit upstream/maint/maint-39
+```
+
+After running the final line, you will have the opportunity to resolve merge
+conflicts. If the conflict is in a large binary file like the ICU4J data jar
+files, you may need to re-generate them.
+
+Remember to prefix your commit message with the ticket number:
+
+```
+git commit -am "ICU-##### Merge branch 'maint/maint-39' into 64-merge-branch"
+git push -u origin 64-merge-branch
+```
+
+As in the Easy Way, you may need to add "DISABLE_JIRA_ISSUE_MATCH=true" and/or
+"ALLOW_MANY_COMMITS=true" to the PR description to silence errors coming from
+the Unicode bot.
+
+Send the PR off for review. As in the Easy Way, **you should use the MERGE COMMIT option in GitHub to land the PR!!**
+
+## Manually Landing PRs
+
+There may be times when a PR is landed manually. This dangerous topic is covered
+in a [separate subpage](manual-land.md).
+
+## Requesting an Exhaustive Test run on a Pull-Request (PR)**
+
+The ICU4C and ICU4J Exhaustive Tests run on the main branch after a pull-request
+has been submitted. They do not run on pull-requests by default as they take 1-2
+hours to run.
+
+However, you can manually request the CI builds to run the exhaustive tests on a
+PR by commenting with the following text:
+
+/azp run CI-Exhaustive
```

Comment:
@rp9-next Surround with a Markdown code fence?  (the "```" before and after a block of code)

Reply:
Thanks! [Done ](https://github.com/unicode-org/icu/pull/1785/commits/4150697514c179419b742e367befb52fe12f06e6)